sweep/sweeper.go

```diff
@@ -996,6 +1019,43 @@ func (s *UtxoSweeper) waitForSpend(outpoint wire.OutPoint,
 	return spendEvent.Cancel, nil
 }
 
+// PendingInputs returns the set of inputs that the UtxoSweeper is currently
+// attempting to sweep. These are sorted in ascending fee rate order.
+func (s *UtxoSweeper) PendingInputs() ([]*PendingInput, error) {
+	respChan := make(chan []*PendingInput, 1)
+	select {
+	case s.pendingSweepsReqs <- &pendingSweepsReq{
+		respChan: respChan,
+	}:
+	case <-s.quit:
+		return nil, ErrSweeperShuttingDown
+	}
+
+	select {
+	case pendingSweeps := <-respChan:
+		return pendingSweeps, nil
+	case <-s.quit:
+		return nil, ErrSweeperShuttingDown
+	}
+}
+
+// handlePendingSweepsReq handles a request to retrieve all pending inputs the
+// UtxoSweeper is attempting to sweep.
+func (s *UtxoSweeper) handlePendingSweepsReq(req *pendingSweepsReq) []*PendingInput {
+	pendingInputs := make([]*PendingInput, 0, len(s.pendingInputs))
+	for _, pendingInput := range s.pendingInputs {
+		pendingInputs = append(pendingInputs, pendingInput)
+	}
+
+	// We'll sort the inputs in ascending fee rate order as a convenience to
+	// the caller.
+	sort.Slice(pendingInputs, func(i, j int) bool {
+		return pendingInputs[i].FeeRate < pendingInputs[j].FeeRate
+	})
```

Comment:
This may be more a thing to do in `rpcserver.go`. It is mainly presentation. (non-blocking)

Reply:
Good point, it's only done within `lncli` now.